package-res/resources/web/pentaho/data/filter/AbstractPropertyFilter.js

```diff
@@ -84,11 +85,9 @@ define([
       O.setConst(this, "value", value);
       O.setConst(this, "property", property);
     },
+
     /**
-     *
-     * @param value
-     * @returns {boolean}
-     * @protected
+     * @inheritdoc
```

Comment:
Running ant-js does not show any docs, because you moved the description to an interface.